### DISCUSSION TITLE
Get-PnPTeamsTeam : Failed to execute Skype backend request GetThreadS2SRequest.

### DISCUSSION THREAD

**Discussion by schertle15** [pnp/powershell#3189] [Q&A]
We try to use Get-PnPTeamsTeam and got an error.
We are connecting like that to the Tenenant:
Connect-PnPOnline -Url $url -ClientId $ClientId -Thumbprint $Thumbprint -Tenant $Tenant -Verbose

This commands are working:

Get-PnPTenantSite

and also

Get-PnPTeamsTeam -Identity "PRJ_M365_Test"

Used Module:
1.12.0     PnP.PowerShell
But it's also not working with the actual version.

NuGet                    2.8.5.208

**Comment by soumyadeep22** [comment]
Yeah, Same issue I am also facing. Did you get any luck on executing? Get-PnPTeamsTeam is giving : Failed to execute backend request. Azure AD app has required one of Group.Read.All permission scope

![image](https://github.com/pnp/powershell/assets/101791799/228f197d-5746-4e87-9a54-52f760140ce3)

**Reply by schertle15** [reply]
No, unfortunately not. Still not working.

**Comment by robterrins** [comment]
I'm also encountering the same issue but also with Add-PnPTeamsChannel and Get-PnPTeamsChannel. 
Did anyone ever find an answer?

**Comment by glasserre-pdw** [comment]
Hello.
We got the same issue with Add-PnPTeamsChannel.
Info : 
PS Version : 5.1
PNP.Powershell : 1.11.0
Script run in an Azure runbook.

**Comment by hajjkh** [comment]
Hi, same issue with command Add-PnPTeamsChannel.

Info :
PS Version : 7
PNP.Powershell : 2.1.1
